### PR TITLE
Feature/ui styling tracker usage

### PR DESCRIPTION
**Pull request by astewes**
-  Adds initial layout for tracker usage metrics - top applications, top programs, top datasets
-  Note: This PR is only for the layout. Placeholder data being used for now, until integrated with the backend API

metrics

Responsive:
responsive

### CODE REVIEW

File: cdap-ui/app/features/tracker/tracker.less [hunk 1]
```diff
@@ -102,6 +107,60 @@ body.state-tracker-home.state-tracker {
       margin-right: 8px;
     }
   }
+
+  div.tracker-metrics {
+    p {
+      line-height: 1;
+      &.metric-heading {
+        color: #333333;
```
Comment (elfenheart) [inline cdap-ui/app/features/tracker/tracker.less]: I think these colors already have some LESS variables

**Comment by elfenheart** [comment]
at 991px:

screen shot 2016-06-19 at 10 04 27 pm

**Comment by ajainarayanan** [comment]
Bamboo build: http://builds.cask.co/browse/CDAP-DRC3876

**Comment by elfenheart** [comment]
👍  LGTM

**Comment by russorat** [comment]
UI LGTM

**Pull request merged**